bazel/README.md

```diff
@@ -876,6 +876,8 @@ To run the tools directly, you must install the correct version of clang. This
 may change over time, check the version of clang in the docker image. You must
 also have 'buildifier' installed from the bazel distribution.
 
+Note that if you run `check_spelling.py` script, it requires `aspell` command.
```

Comment:
Perhaps

> Note that if you run the `check_spelling.py` script you will need to have `aspell` installed

this still doesnt quite capture it because in reality you need to have `aspell` + the same dictionaries/locale as the envoy ci - for now tho i think this is ok